decidim-accountability/decidim-accountability.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |s|
     "homepage_uri" => "https://decidim.org",
     "source_code_uri" => "https://github.com/decidim/decidim"
   }
-  s.required_ruby_version = ">= 3.1"
+  s.required_ruby_version = ">= 3.2"
```

Comment:
Semantic lock to 3.2 will be done as a stand alone PR, so that we can easily backport to other maintained versions.